config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
-            href: "https://www.citizensadvice.org.uk/scotland/benefits/universal-credit/"
-            show_to_nations:
-              - Scotland
-          - id: "0056"
-            text: "Get information on applying for Universal Credit (Citizens Advice)"
-            href: "https://www.citizensadvice.org.uk/wales/benefits/universal-credit/"
-            show_to_nations:
-              - Wales
-          - id: "0057"
-            text: Get help from Citizens Advice
-            href: "https://www.citizensadvice.org.uk/about-us/northern-ireland/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0058"
-            text: "Get advice on benefits (Advice NI)"
-            href: "https://www.adviceni.net/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0059"
-            text: "Contact Working Wales if you’ve been made redundant"
-            href: "https://workingwales.gov.wales/"
-            show_to_nations:
-              - Wales
-          - id: "0060"
-            text: "Contact Citizens Advice Wales"
-            href: "https://www.citizensadvice.org.uk/wales/about-us/contact-us/contact-us/contact-us/"
-            show_to_nations:
-              - Wales
-          - id: "0061"
-            text: "Check if you’re eligible for the Discretionary Assistance Fund (Welsh Government)"
-            href: "https://gov.wales/discretionary-assistance-fund-daf"
-            show_to_nations:
-              - Wales
-          - id: "0062"
-            text: "Get help if you’ve been made redundant (mygov.scot)"
-            href: "https://www.mygov.scot/help-redundancy/"
-            show_to_nations:
-              - Scotland
-          - id: "0063"
-            text: "Find help with benefits (mygov.scot)"
-            href: "https://www.mygov.scot/benefits-support/"
-            show_to_nations:
-              - Scotland
+        - id: '0045'
+          text: What to do if you’ve lost your job
+          href: https://www.gov.uk/guidance/coronavirus-covid-19-what-to-do-if-you-were-employed-and-have-lost-your-job
+        - id: '0046'
+          text: What to do if you’ve been laid-off
+          href: https://www.gov.uk/lay-offs-short-timeworking
+        - id: '0047'
+          text: Find out about your rights if you’ve been dismissed
+          href: https://www.gov.uk/dismissal
+        - id: '0048'
+          text: What to do if you’ve been furloughed
+          href: https://www.gov.uk/guidance/coronavirus-covid-19-what-to-do-if-youre-employed-and-cannot-work
+        - id: '0049'
+          text: What to do if your employer has told you not to work (Citizens Advice)
+          href: https://www.citizensadvice.org.uk/work/coronavirus-if-your-employer-has-told-you-not-to-work/
+          show_to_nations:
+          - England
+        - id: '0050'
+          text: What to do if your employer has told you not to work (Citizens Advice
+            Scotland)
```

Comment:
```suggestion
          text: |
            What to do if your employer has told you not to work (Citizens Advice
            Scotland)
```

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216